cloud/pkg/controller/config/buffer.go

```diff
@@ -18,32 +18,63 @@ var QueryConfigMapBuffer int
 // QuerySecretBuffer is the size of channel which save query secret message from edge
 var QuerySecretBuffer int
 
+// PodEventBuffer is the size of channel which save pod event from k8s
+var PodEventBuffer int
+
+// ConfigMapEventBuffer is the size of channel which save configmap event from k8s
+var ConfigMapEventBuffer int
+
+// SecretEventBuffer is the size of channel which save secret event from k8s
+var SecretEventBuffer int
+
 func init() {
 	if psb, err := config.CONFIG.GetValue("controller.buffer.update-pod-status").ToInt(); err != nil {
 		UpdatePodStatusBuffer = constants.DefaultUpdatePodStatusBuffer
 	} else {
 		UpdatePodStatusBuffer = psb
 	}
-	log.LOGGER.Infof("update pod status buffer: %d", UpdatePodStatusBuffer)
+	log.LOGGER.Infof("controller update pod status buffer: %d", UpdatePodStatusBuffer)
 
 	if nsb, err := config.CONFIG.GetValue("controller.buffer.update-node-status").ToInt(); err != nil {
 		UpdateNodeStatusBuffer = constants.DefaultUpdateNodeStatusBuffer
 	} else {
 		UpdateNodeStatusBuffer = nsb
 	}
-	log.LOGGER.Infof("Update node status buffer: %d", UpdateNodeStatusBuffer)
+	log.LOGGER.Infof("controller Update node status buffer: %d", UpdateNodeStatusBuffer)
 
 	if qcb, err := config.CONFIG.GetValue("controller.buffer.query-configmap").ToInt(); err != nil {
 		QueryConfigMapBuffer = constants.DefaultQueryConfigMapBuffer
 	} else {
 		QueryConfigMapBuffer = qcb
 	}
-	log.LOGGER.Infof("query config map buffer: %d", QueryConfigMapBuffer)
+	log.LOGGER.Infof("controller query config map buffer: %d", QueryConfigMapBuffer)
 
 	if qsb, err := config.CONFIG.GetValue("controller.buffer.query-secret").ToInt(); err != nil {
 		QuerySecretBuffer = constants.DefaultQuerySecretBuffer
 	} else {
 		QuerySecretBuffer = qsb
 	}
-	log.LOGGER.Infof("query secret buffer: %d", QuerySecretBuffer)
+	log.LOGGER.Infof("controller query secret buffer: %d", QuerySecretBuffer)
+
+	if peb, err := config.CONFIG.GetValue("controller.pod-event-buffer").ToInt(); err != nil {
```

Comment:
The following modifications are welcome:
Replace "controller.pod-event-buffer" with "controller.buffer.pod-event"
Replace "controller.configmap-event-buffer" with "controller.buffer.configmap-event"
Replace "controller.secret-event-buffer" with "controller.buffer.secret-event"

@sids-b @kadisi How about your ideas?

Reply:
Agree with @edisonxiang

Reply:
ok